src/main/java/org/apache/tomcat/jni/SSL.java

```diff
@@ -234,6 +234,9 @@
     public static final int SSL_SELECTOR_FAILURE_NO_ADVERTISE = 0;
     public static final int SSL_SELECTOR_FAILURE_CHOOSE_MY_LAST_PROTOCOL = 1;
 
+    public static final int SSL_ST_CONNECT = 0x1000;
```

Comment:
yeah maybe... I just don't want to change too many things as we may get into merge trouble once we change too many things and netty to upgrade to a new tcnative version for our fork